src/components/validationStatus.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div>
+    <div class="validation-container" v-if="toggleValidationPanel">
+      <span class="validation-container__defaultMessage" v-if="!numberOfValidationErrors">no problems to report</span>
+      <div class="validation-container__list"  v-for="error in errorList" :key="error.id">
+        <span class="validation-container__list--id">{{ error.id }}</span>
+        <span class="validation-container__list--message">
+          <span class="validation-container__list--key">{{ error.errorKey }}</span>
+          <span>{{ error.message }}</span>
+        </span>
+        <span class="validation-container__list--errorCategory"
+              :style="[ error.category === 'error'
+              ? { 'background-color': `${ errorColor }` }
+              : { 'background-color': `${ warningColor }` }]">
+              {{ error.category }}
+        </span>
+      </div>
+    </div>
+     <div class="statusBar-container" @click="toggleValidationPanel = !toggleValidationPanel">
+      <span class="statusBar-container__status-text">Problems {{ numberOfValidationErrors }}</span>
+      <font-awesome-icon class="statusBar-container__status-icon" :style="{ color: statusColor }" :icon="statusIcon" />
+      <font-awesome-icon class="statusBar-container__status-ellipsis" :icon="ellipsisIcon" />
```

Comment:
I think you should stick to kebab-case here too. Above you have `validation-bar-container`, but then you break this rule here and use `statusBar-container` instead of `status-bar-container` or `statusbar-container`.